pkg/features/kube_features.go

```diff
@@ -371,7 +365,6 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	PodPriority:                                 {Default: true, PreRelease: utilfeature.Beta},
 	EnableEquivalenceClassCache:                 {Default: false, PreRelease: utilfeature.Alpha},
 	TaintNodesByCondition:                       {Default: true, PreRelease: utilfeature.Beta},
-	MountPropagation:                            {Default: true, PreRelease: utilfeature.Beta},
```

Comment:
should be changed to GA and defaulted true, right?

Reply:
@kubernetes/sig-architecture-pr-reviews - does the expected lifecycle of a feature gate stop honoring the gate being disabled once it reaches GA?

Reply:
I didn't find anything about removing feature gate in community repo. Did I miss it?

I found issue #46404 that suggests we should remove all gate checks, report the feature as GA for 2 releases and then remove it.

#65862 followed this approach, but they remove the gate checks when removing the feature, not when it was declared GA.

Reply:
That's not completely true. Advanced Audit Logging is actually moves to GA, I remove all gate checks and remove also old pipeline that is not possible to execute without disabling Advanced Audit Logging (because it in GA)

Reply:
Yes, I think the expectation should be that once a feature is GA it cannot be disabled, even though we need to leave the gate in place.

Reply:
See the thread at https://groups.google.com/d/msg/kubernetes-sig-architecture/qJeJ7ANYzS0/0KnJgirWDgAJ

Promoting to GA and announcing deprecation of the gate in 1.12, then removing it and the checks of it in 1.13 would be consistent with our deprecation policy.